UCLA Radio/UCLA Radio/menu/NotificationShowCell.swift

```diff
@@ -72,6 +74,26 @@ class NotificationShowCell: UITableViewCell, ScheduleShowCellDelegate {
     
     // MARK: - Layout
     
+    func styleDeleteButton() {
+        for subview in subviews {
+            // UITableViewCellDeleteConfirmationView is in a private API, so we have to check it by converting the type to string
+            if String(describing: type(of: subview)) == "UITableViewCellDeleteConfirmationView" {
```

Comment:
cool -- generally hidden APIs are dangerous but this is a good use of it, doesn't matter much if it breaks